src/emqx_sys_mon.erl

```diff
@@ -66,7 +66,7 @@ parse_opt(Opts) ->
     parse_opt(Opts, []).
 parse_opt([], Acc) ->
     Acc;
-parse_opt([{long_gc, false}|Opts], Acc) ->
+parse_opt([{long_gc, 0}|Opts], Acc) ->
```

Comment:
When long_gc value is 0, it equals disable long_gc, you can refer [here](https://github.com/erlang/otp/blob/feb45017da36be78d4c5784d758ede619fa7bfd3/erts/emulator/beam/erl_gc.c#L421)